ComponentKit/Core/CKComponentLayout.h

```diff
@@ -62,14 +62,21 @@ NSSet *CKMountComponentLayout(const CKComponentLayout &layout,
                               CKComponent *supercomponent);
 
 /**
- This exists as a safe alternative to layoutThatFits:parentSize: by guarding against nil components
- @param component The component being laid out
- @param sizeRange The size range for laying out the component
- @param parentSize The size of the parent of the component being laid out
+ Safely computes the layout of the given top-level component by guarding against nil components.
+ @param component The top-level component to compute the layout for.
+ @param sizeRange The size range to compute the component layout within.
  */
-CKComponentLayout CKComponentComputeLayout(CKComponent *component,
+CKComponentLayout CKComputeTopLevelComponentLayout(CKComponent *component, const CKSizeRange &sizeRange);
```

Comment:
I mildly prefer Root to TopLevel since the former is already a componentkit concept.